src/views/Page404/index.jsx

```diff
@@ -1,7 +1,7 @@
 import { Typography, Grid } from '@mui/material';
 import mascot from './mascot.svg';
 import ScottieDog from './components/ScottieDog';
-
+import styles from './Page404.module.css';
 const message = "Hmmm... We're not quite sure what you're looking for";
 
 const styles2 = {
```

Comment:
You should be able to convert all of this to external CSS.  Everything in here is CSS, so it should work fine to convert.  That way, we can get rid of the `styles2 = ` line.